first_hour_on_vwb/working_with_data_collections.ipynb

```diff
@@ -0,0 +1,603 @@
+    "        self.input_name = wu.TextInputWidget(\"<WORKSPACE_NAME>\",\"Workspace Name:\").get()\n",
+    "        self.input_description = wu.TextInputWidget(\"<DESCRIPTION>\",\"Description:\").get()\n",
+    "        self.input_workspace_id = wu.TextInputWidget(\"<WORKSPACE_ID>\",\"Workspace ID:\").get()\n",
+    "        self.output_workspace_id = widgets.Text()\n",
+    "        self.output_workspace_id.value = self.input_workspace_id.value\n",
+    "        self.button = wu.StyledButton('Create workspace','Click to create a new workspace','plus').get()\n",
+    "        self.button.on_click(self.create_workspace)\n",
+    "        self.output = widgets.Output()\n",
+    "        self.vb = widgets.VBox(\n",
+    "            children = [self.label, self.warning,\n",
+    "                        self.input_name, self.input_description,\n",
+    "                        self.input_workspace_id,\n",
+    "                        self.button, self.output],\n",
+    "            layout = wu.vbox_layout)\n",
+    "        \n",
+    "    def get_workspace_id(self):\n",
+    "        return self.input_workspace_id.value\n",
+    "\n",
+    "    def create_workspace(self,b):\n",
+    "        with self.output:\n",
+    "            commandList = [\n",
+    "                \"terra\", \"workspace\", \"create\",\n",
+    "                f\"--id={self.input_workspace_id.value.strip()}\",\n",
+    "                f\"--description=\\\"{self.input_description.value}\\\"\",\n",
+    "                f\"--name={self.input_name.value.strip()}\",\n",
+    "            ]\n",
+    "\n",
+    "            print('Running command to create workspace...')\n",
+    "            print('\\n'.join(commandList))\n",
+    "            print('')\n",
+    "            print(\"Your workspace will be ready in less than one minute...\")\n",
+    "            \n",
+    "            result = subprocess.run(commandList, capture_output = True, text = True, timeout=180, check=True)\n",
+    "            print(result.stderr) if not result.stdout else print(result.stdout)\n",
+    "            \n",
+    "            # refreshWorkspaceContextCommand = \"terra workspace list\"\n",
+    "            # result = subprocess.run(refreshWorkspaceContextCommand, shell = True, capture_output = True, text = True, timeout=30, check=True)\n",
+    "\n",
+    "create_ws_widget = CreateWorkspaceWidget()\n",
+    "display(create_ws_widget.vb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Convert new workspace to data collection\n",
+    "<a id=\"convert-to-dc\"></a>\n",
+    "\n",
+    "Now you'll convert your newly created workspace, to which you have added resources, into a data collection which can be shared with others and added to other workspaces. \n",
```

Comment:
I think this section needs a bit on that 'converting to a data collection' doesn't yet mean that they will see their new DC in the data catalog yet— it won't show up there until they publish a version, which they'll do next.

Reply:
fixed!